packages/ember-glimmer/tests/integration/helpers/log-test.js

```diff
@@ -0,0 +1,60 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import { get } from 'ember-metal/property_get';
+import Logger from 'ember-metal/logger';
+
+var originalLog, logCalls;
+
+moduleFor('Helpers test: {{log}}', class extends RenderingTest {
```

Comment:
I can certainly add this in, but the description for the log helper said as an exception here to only test initial render.